SystemManager/Inc/SystemManager.hpp

```diff
@@ -10,6 +10,10 @@
 #include "independent_watchdog.h"
 #include "ZP_D_PWMChannel.hpp"
 
+#define SBUS_THRESHOLD 25
+#define SBUS_MIN 0
+#define SBUS_MAX 100
+
```

Comment:
Why are these defined here instead of the SBUS class?

Reply:
either these are in SM, or in SBUS and exposed to SM, but it sounds like the only component that needs these values is SM so i see arguments for keeping it here and arguments for putting it in the sbus driver.